ops/docker-compose.yml

```diff
@@ -34,6 +34,7 @@ services:
       context: ../nutrinova-api/
     environment:
       - ConnectionStrings__DefaultConnection=Host=postgres;Port=5432;Database=${POSTGRES_DB};Username=${POSTGRES_USER};Password={POSTGRES_PASSWORD}
+      - USDA_API_KEY={USDA_API_KEY}
```

Comment:
I think you'll need to add it to the github action to pass it to the docker compose

Reply:
Which action?

Reply:
deploy, youll see them all at the top